docs/docs/development.md

```diff
@@ -35,8 +35,8 @@ filename = os.path.expanduser('~/.minikube/profiles/minikube/config.json')
 with open(filename) as f:
```

Comment:
How do I run this? Simple copy and paste on the terminal?

Reply:
Yes, simply paste in terminal